mne/viz/_3d.py

```diff
@@ -682,3 +683,79 @@ def plot_sparse_source_estimates(src, stcs, colors=None, linewidth=2,
     surface.actor.property.shading = True
 
     return surface
+
+
+def plot_dipoles(dipoles, fwd, subject, subjects_dir, bgcolor=(1, ) * 3,
```

Comment:
No, not yet :/ 
But I'm thinking is it not better to remove it from this folder (rap_music) and put it on dipole.py, then we can have dip.plot_dipole(...)? @Eric89GXL